addons/hr/security/hr_security.xml

```diff
@@ -28,9 +28,8 @@
     <record id="hr_employee_comp_rule" model="ir.rule">
         <field name="name">Employee multi company rule</field>
         <field name="model_id" ref="model_hr_employee"/>
-        <field name="domain_force">['|', '|', '|',
+        <field name="domain_force">['|', '|',
             ('company_id', 'in', company_ids + [False]),
-            ('parent_id.user_id', '=', user.id),
```

Comment:
Why do you remove this ? 

This behavior was intended in the context of this task: https://github.com/odoo-dev/odoo/commit/abd909498e4fd519ff8431ea9d7d09da4fc7c59a

Reply:
Hello @tivisse 

Suppose the employee and the manager work for different companies
if you log in through the manager with its company, then this rule becomes true so we can see the employee in the Org view

Reply:
Suppose a BE employee is manager of a IN employee, you would like to see him on the Org Chart right ?

Reply:
Hello @tivisse 
Yes, we would like to see them but if we are only logged in with the BE company, we should not show records for the other company.

Reply:
Discussed with GMF, he'll validate that on testing. 

So keep it that way for now :)

Thanks.